### DISCUSSION TITLE
Can i run sb recorder in UC mode?

### DISCUSSION THREAD

**Discussion by STARKILLER-1** [seleniumbase/SeleniumBase#2264] [Q&A] [answered]
When i'm playing Playback Cloudflare activates. Recorder would be brilliant with uc mode. And can I add cookie to recorder?

**Comment by STARKILLER-1** [comment, marked as answer]
UPD: I replaced `self.open` with `self.get` and deleted `self.open_if_not_url` in `new_recording.py ` and capcha was gone. But cookie question is still a question  =)
The more I learn about SeleniumBase, the more I like it

**Reply by mdmintz** [reply]
You may need to make some modifications to your recorded scripts before playing them back in UC Mode. The Recorder is designed specifically for regular SeleniumBase, not special modes like UC Mode.

**Reply by STARKILLER-1** [reply]
And same about cookies? Because some actions can be done only as authorized user.

**Reply by mdmintz** [reply]
The Recorder isn't recording anything related to cookies. Only browser actions. If you need more than the recorded script, you may need to add in additional lines of code to your recorded script.

**Reply by STARKILLER-1** [reply]
> The Recorder isn't recording anything related to cookies. Only browser actions. If you need more than the recorded script, you may need to add in additional lines of code to your recorded script.

Nvm, Google Chrome released record function, I can resolve my cookie issue using Chrome record function. Thank you!
